src/main.py

```diff
@@ -1,11 +0,0 @@
-from src.constants import db_path, external_funds_folder
-from src.dispatcher import run_tasks
-from src.setup import init
-
-
-def main():
-    db = init(external_funds_folder=external_funds_folder, db_path=db_path)
```

Comment:
I would make both `external_funds_folder` and `db_path` into pydantic attributes, e.g. on a `TaskRunner` class (or similar).